packages/core/src/carousel/Carousel.tsx

```diff
@@ -73,7 +84,14 @@ const CarouselRoot = ({
         firstVisibleIndex,
         lastVisibleIndex,
         autoSlideDelay,
+        isSwipeDragDisabled: isSwipeDragDisabled ?? false,
```

Comment:
Consider using a more explicit default value for clarity

Reply:
This is a flag that has only two possible values `true` or `false`. Based on discussions it's better to have `false` as default.